api/provider/aws/environment_scaler.go

```diff
@@ -1,14 +1,758 @@
 package aws
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"sort"
+	"strconv"
+
+	"github.com/zpatrick/go-bytesize"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/api/provider"
+	awsc "github.com/quintilesims/layer0/common/aws"
+	"github.com/quintilesims/layer0/common/config"
+	"github.com/quintilesims/layer0/common/errors"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+var defaultPorts = []int{
+	22,
+	2376,
+	2375,
+	51678,
+	51679,
+}
+
+// https://aws.amazon.com/ec2/instance-types/
+var InstanceSizes = map[string]bytesize.Bytesize{
```

Comment:
Consider putting this in a separate file in the same `aws` package

Reply:
This should be changed to `InstanceTypes`, and don't we need need to track CPU per instance type as well?

Reply:
@zpatrick Updating for these things; do you feel alright about `instanceSpecs` instead of `InstanceTypes`?